sdk/stable-layout/src/stable_vec.rs

```diff
@@ -56,13 +52,13 @@ impl<T> StableVec<T> {
 
 impl<T> AsRef<[T]> for StableVec<T> {
     fn as_ref(&self) -> &[T] {
-        self
-    }
+        self.deref()
+    } // === is this right?
```

Comment:
Whoops, left in a note by accident. Still, the question stands :). Deref and as_ref are .. fairly close in meaning I guess, and I think identical for our purposes here (?)

Reply:
I missed your comment before submitting my review, but did you hit errors just returning `&self` or `self` like before? I believe that should be sufficient given the struct implements the `Deref` trait on L70 below.

Reply:
Ah, good question. Maybe that's unnecessary. Don't recall if I hit an error or just did it because it looked right to me. I'll have to check.